Point to Chromium-BiDi milestones (#3225)
GoogleChrome/developer.chrome.com: site/en/blog/webdriver-bidi/index.md

Another important aspect of BiDi is Web Platform Tests. Currently they cover WebDriver, and de-facto is a source-of-truth for all the implementations. It’s a cornerstone for cross-browser protocol implementation, and intended to be run and to be passed against any implementation.

## What is the current state?

> The current state of our implementation and the specification can be tracked here: [WebDriver BiDi prototyping progress](https://github.com/GoogleChromeLabs/chromium-bidi/milestones)

Reviewer: ```suggestion
To track the current state of our implementation and the specification, see the [WebDriver BiDi prototyping progress](https://github.com/GoogleChromeLabs/chromium-bidi/milestones).
```
